build_projects/dotnet-host-build/project.json

```diff
@@ -6,7 +6,7 @@
   },
   "dependencies": {
     "NETStandard.Library": "1.6.0-rc3-24201-00",
-    "Microsoft.NETCore.Runtime": "1.0.2-rc3-24201-00",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2-rc3-24201-00",
```

Comment:
We should simply move this to reference Microsoft.NETCore.App, there's no reason to reference these packages directly here.